src/modules/ToDoList/ToDoList.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import '../../styles/ToDoListStyles/ToDoList.css'
+
+import ToDoTask from "./ToDoTask";
+
+
+class ToDoList extends React.Component {
+    constructor(props) {
+        super(props);
+
+        console.log("props.tasks: ", this.props.tasks);
+
+        this.state = {
+            tasks: this.props.tasks,
+            inputValue: '',
+            editValue: ''
+        };
+    }
+    static getDerivedStateFromProps(props, state){
+        if(props.tasks!== state.tasks){
+            return {tasks: props.tasks };
+            // this.classMethod();
+        }
+    }
+
+    componentWillReceiveProps(nextProps){
+        if(nextProps.tasks!==this.props.tasks){
+            this.setState({tasks: nextProps.tasks });
+            // this.classMethod();
+        }
+    }
+
+    removeTask = (index) => {
+        let { tasks } = this.state;
```

Comment:
const, not let, and please check all your methods on that, and replace let with const where it's necessary